samples/test/sample.test.js

```diff
@@ -75,6 +75,22 @@ describe('quickstart sample tests', () => {
     await file.delete();
   });
 
+  it('should export asset relationships to specified path', async () => {
+    const dumpFilePath = `gs://${bucketName}/my-relationships.txt`;
```

Comment:
what would happen if two test runs were happening at the same time, wouldn't they collide on this bucket?

I think ideally we'd suffix the file with a `uuid`:

```
gs://${bucketName}/${Date.now()}-{uuid}.txt
```

We would then also need a cleanup step so that we don't leak an infinite number of files.

What you can do is list the `bucketName`, parse the file names, and delete any with an older timestamp than a few hours.

Reply:
It is an atomic op, so the later will overwrite the previous one.  So we only have one file for the test.  Thanks

Reply:
my concern would be that two test runs are happening in conjunction, e.g., two PRs are merged closely to each other, or two PRs are opened at once.

If this happens `await file.delete();` running in test run `A` could break test run `B`, which is waiting on the contents of `my-relationships.txt` which will never exist.

I think this could potentially lead to test flakes.

Reply:
I see your point.  
Add uuid to the file names. 
In each test, we will delete the file created.  So no need to do the list anymore, PTAL.  Thanks